src/modules/sessions/sessions.route.ts

```diff
@@ -170,9 +181,19 @@ sessionRouter.post(
   controller.createSession
 );
 
+/**
+ * PATCH /sessions
+ * @typedef {object} IUpdateMainSession
+ * @tags sessions
+ * @param {UpdateMainSession} request.body.required
+ * @summary Update Main Session
+ * @security BearerAuth
+ * @return {UpdateMainSession}
+ *
+ */
 sessionRouter.patch(
   '/',
-  verifyAuthToken,
+  // verifyAuthToken,
```

Comment:
change this into verifyAdminAuthToken so we can assure that FOUNDER type can update user